File: Find Last Digit/FindLastDigit.java
public class FindLastDigit {
    public static void main(String[] args) {
        java.util.Scanner sc = new java.util.Scanner(System.in);
        System.out.print("Enter a number: ");
        int n = sc.nextInt();
        int lastDigit = Math.abs(n) % 10;
        System.out.println("The last digit is: " + lastDigit);
    }
}

File: Find Last Digit/find_last_digit.c
#include <stdio.h>
#include <stdlib.h>
int main() {
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);
    int last_digit = abs(n) % 10;
    printf("The last digit is: %d\n", last_digit);
    return 0;
}
